pkgs/development/tools/tabnine/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchurl }:
+
+let
+  hashes = {
+    x86_64-linux   = "caa86a00d0ede8485bf855276ac1b2e411a5e99de612696b137ea3519a8e4967";
+    x86_64-darwin  = "eb5ceb534629dcf5d57640bf672ea0d94ac02e264209ab3a29e6bda8a44d3b8f";
+  };
+in
+stdenv.mkDerivation rec {
+  name = "tabnine-${version}";
+  version = "1.0.12";
+
+  platformString =
+    if stdenv.isDarwin then "x86_64-apple-darwin"
+    else if stdenv.isLinux then "x86_64-unknown-linux-gnu"
+    else throw "unsupported platform";
+
+  src = fetchurl {
+    url = "https://update.tabnine.com/${version}/${platformString}/TabNine";
+    sha256 = hashes.${stdenv.hostPlatform.system} or (throw "Unsupported system: ${stdenv.hostPlatform.system}");
+  };
+
+  phases = "installPhase";
```

Comment:
Don't set phases directly, instead you can use `unpackPhase = "true";` to disable the unpack phase.